### PR TITLE
Match `AudioOcarina_CreateCustomSequence` (code_8019AF00.c OK)

### PR DESCRIPTION
**Pull request by engineer124**

### CODE REVIEW

**Review by sonicdcer — APPROVED** [review]
Excellent work!

File: src/audio/code_8019AF00.c
Comment (AngheloAlf) [inline src/audio/code_8019AF00.c:1]: So, what was it? variable re-use?

**Review by AngheloAlf — APPROVED** [review]

File: src/audio/code_8019AF00.c
Comment (isghj5) [reply src/audio/code_8019AF00.c:1]: Yeah I don't see it either, it wasn't just removing the if1 and re-order stack was it?

Comment (engineer124) [reply src/audio/code_8019AF00.c:1]: Main thing was combining `temp_a0` and `temp_t4` as one reused temp (`temp`)

**Pull request merged**